src/components/StartupModal/StartupModal.tsx

```diff
@@ -1,256 +1,99 @@
-            />
-            <Button
-              small
-              label="Player1"
-              onClick={e => {
-                hanldeTabClick(e, "player1");
-                process.env && setDevNodeTypes("player1");
-              }}
-            />
-            <Button
-              small
-              label="Player2"
-              onClick={e => {
-                hanldeTabClick(e, "player2");
-                process.env && setDevNodeTypes("player2");
-              }}
-            />
-          </div>
-          <div id="Dealer" />
-          {nodesToInput
-            .filter(node => node.id === nodeType)
-            .map((node, key) => {
-              return (
-                <div key={key}>
-                  {/* Label*/}
-                  <div
-                    css={css`
-                      color: ${theme.moon.colors.text};
-                      padding: 1rem 0 0.5rem 0;
-                      font-size: 0.875rem;
-                    `}
-                  >
-                    {node.name}
-                  </div>
-                  {/* Input field */}
-                  <input
-                    css={css`
-                      background: none;
-                      border: 1px solid ${theme.moon.colors.primary};
-                      color: white;
-                      font-family: sans-serif;
-                      font-weight: 500;
-                      text-align: center;
-                      padding: 0.5rem 0.25rem;
-                      width: 100%;
+    <Modal>
+      {/* TabsWrapper */}
+      <Tabs css={tabsStyle}>
+        <TabList>
+          <Tab>Table List</Tab>
+          <Tab>Custom IP</Tab>
+        </TabList>
 
-                      &:focus {
-                        border: 1px solid ${theme.moon.colors.accent};
-                      }
-                    `}
-                    name={node.name}
-                    placeholder={`192.168.101.234`}
-                    defaultValue={process.env ? node.devAddress : ""}
-                    onChange={e => {
-                      setNodes({
-                        ...nodes,
-                        [node.name]: e.target.value
-                      });
-                    }}
-                  />
-                </div>
-              );
-            })}
-          <div
-            css={css`
-              text-align: center;
-              padding-top: 2rem;
-            `}
-          >
-            <Button
-              label="Set Nodes"
-              disabled={!canSetNodes}
-              onClick={e => {
-                e.preventDefault();
-                handleSubmit();
-              }}
-            />
-          </div>
-        </form>
-      </div>
-    </div>
+        <TabPanel>
+          <TableSelect dispatch={dispatch} state={state} />
+        </TabPanel>
+        <TabPanel>
+          <CustomIP />
+        </TabPanel>
+      </Tabs>
+    </Modal>
   );
 };
 
-export default Modal;
+export default StartupModal;
```

Comment:
Thanks for the input.

From the [docs](https://reactjs.org/docs/react-component.html#displayname):

> The `displayName` string is used in debugging messages. Usually, you don’t need to set it explicitly because it’s inferred from the name of the function or class that defines the component. [...]

What benefit do we gain here if we set `displayName` explicitly?